Introduction.md

```diff
@@ -32,6 +32,22 @@ There are many different reasons for companies to contribute to existing OSS pro
 
 ## Maturity levels
 
-TODO: Describe maturity model of engaging with open source, from passive use, over contributions to existing projects, starting own project, to strategic cocreation across company boundaries. See [issue #8](https://github.com/Open-Source-Compliance/outbound-oss/issues/8).
+Corporate adoption of open source can typically be described using a model of maturity levels. These levels characterize how, with growing maturity, open source is used in an increasingly effective fashion to drive value and realize business needs. How outbound open source is handled is a distinguishing factor between the different levels of maturity. A critical factor is the insight that influencing the open source ecosystem is mainly done by participation and doing work as contributions to open source projects.
 
-Mozilla archetypes?
+A typical maturity model of corporate open source adoption will look like this (see for example [Haddad: Open Source Program Offices](https://www.linkedin.com/pulse/open-source-program-offices-primer-organizational-roles-haddad)):
+
+0. Denial - No or unconscious use of open source
+1. Consumption - Passive use of open source software
+2. Participation - Engagement with open source community
+3. Contribution - Pragmatic contributions to open source projects
+4. Leadership - Strategic involvement with open source to drive business value
+
+To advance from one level to another it requires certain initiatives and structural and organizational elements.
+
+Going from consumption to participation will start with informal engagement and low-effort activities such as reporting bugs in upstream projects driven by technical needs. Decisions about open source contributions will be ad-hoc and be taken for individual cases.
+
+Establishing decision making processes and formalizing contribution policies will lead to the next level. A typical step on this level is to establish an Open Source Program Office to support open source engagement and maintain open source strategy and process.
+
+On the leadership level, processes are mature and have to be scaled. Corresponding tooling needs to be made available. Own projects with the goal to create new open source communities will be started. This will typically come with leveraging open source foundation to enable cross-company collaboration to strategically use open source to accelerate creating business value.
```

Comment:
IMO, not only on the leadership level processes are mature (this could happen but is not necessarily the case). On consumption level, for example, inbound processes could also be mature though a company on that level pretty much does nothing else but consuming open source.

Reply:
You are right. Maybe we should qualify this as **contribution** processes on this level.